pkg/os/volume/api.go

```diff
@@ -149,92 +177,117 @@ func (VolumeAPI) UnmountVolume(volumeID, path string) error {
 		return err
 	}
 
-	cmd := `Get-Volume -UniqueId "$Env:volumeID" | Get-Partition | Remove-PartitionAccessPath -AccessPath $Env:mountpath`
-	cmdEnv := []string{}
-	cmdEnv = append(cmdEnv, fmt.Sprintf("volumeID=%s", volumeID))
-	cmdEnv = append(cmdEnv, fmt.Sprintf("mountpath=%s", path))
-	out, err := utils.RunPowershellCmd(cmd, cmdEnv...)
-
+	mountPoint := path
+	if !strings.HasSuffix(mountPoint, "\\") {
+		mountPoint += "\\"
+	}
+	utf16MountPath, _ := windows.UTF16PtrFromString(mountPoint)
+	err := windows.DeleteVolumeMountPoint(utf16MountPath)
```

Comment:
is this identical to `Remove-PartitionAccessPath` command?

Reply:
I believe so.
The `Add-PartitionAccessPath` should be identical to `SetVolumeMountPoint` Win32 API call and `Remove-PartitionAcessPath` should be identical to `DeleteVolumeMountPoint` Win32 API call.
I tried to call `getTarget` (with calling to `GetVolumeNameForVolumeMountPoint`) after `Add-PartitionAccessPath`, then call `DeleteVolumeMountPoint` without having issues in a local testing.